client/src/components/BottomPanel/components/ChromosomeMap/ChromosomeMap.tsx

```diff
@@ -109,6 +77,52 @@ export const ChromosomeMap = () => {
 
   const totalBPAtScale = (totalBasePairs * binSize) / 1_000; // this gives us a scale in kb
 
+  // Find the selected gene info and calculate its position
+  const selectedGeneInfo = useMemo(() => {
+    for (const q of coverageQueries) {
+      if (q.data?.geneInfo) {
+        const gene = q.data.geneInfo.find(
+          (g) => g.geneName.toLowerCase() === selectedGene.toLowerCase()
+        );
+        if (gene) return gene;
+      }
+    }
+    return null;
+  }, [coverageQueries, selectedGene]);
+  useEffect(() => {
+    // Calculate scroll position for the selected gene
+    const scrollToGene = () => {
+      if (
+        !scrollContainerRef.current ||
+        !selectedGeneInfo ||
+        !startBasePair ||
+        !binSize
+      ) {
+        return;
+      }
+
+      // Calculate the pixel position relative to the start of the coverage data
+      const relativePosition = selectedGeneInfo.geneStart - startBasePair;
+      const pixelPosition = (relativePosition / binSize) * BAR_WIDTH;
+      // Subtracting 20px to place the gene nicely in the viewport
+      const scrollPosition = Math.max(0, pixelPosition - 20);
```

Comment:
I wonder if we could add an `id` attribute to each gene annotation svg element and just scroll to that element with `scrollIntoView()`?

Somewhere here?
https://github.com/chanzuckerberg/single-cell-explorer/blob/cdb3ccb3375b4882da108affb3b831f4731f8a07/client/src/components/BottomPanel/components/GeneMap/components/GeneMapSVG/GeneMapSVG.tsx#L248